src/gap.c

```diff
@@ -815,11 +816,17 @@ static Obj FuncGASMAN(Obj self, Obj args)
             SyMsgsFlagBags = (SyMsgsFlagBags + 1) % 3;
         }
 
+        /* or disable workspace size limits                                */
+        else if (streq(CONST_CSTR_STRING(cmd), "disablelimit")) {
+            SyGasmanDisableMemoryLimits();
```

Comment:
Personal opinion (not a blocker for this PR, just a remark): I am not very fond of adding more subcommands to `GASMAN`. Why not just add this as a separate function, say `DISABLE_MEMORY_LIMITS`? On the long run, I'd prefer to remove more GASMAN subcommands, too (or at least add alternatives, and only keep the subcommands for backwards compatibility. E.g. instead of the `collect`/`partial` subcommands, one should use the `CollectGarbage` function; and perhaps we should just turn that into a kernel function (so that it doesn't go through `GASMAN`, as it does right now).